index.js

```diff
@@ -3,18 +3,10 @@
 const fs = require('fs');
 const path = require('path');
 const d3 = require('d3-queue');
-const schema = require('protocol-buffers-schema');
-const Pbf = require('pbf');
-const Compile = require('pbf/compile');
+const util = require('./lib/util');
+const generateBuffer = require('./lib/generateBuffer');
 
-const proto_mvt = schema.parse(fs.readFileSync('vector-tile-spec/2.1/vector_tile.proto', 'utf8'));
-const mvt = Compile(proto_mvt);
-
-function generateBuffer(json) {
-  const pbf = new Pbf();
-  mvt.Tile.write(json, pbf);
-  return new Buffer(pbf.finish());
-};
+const eight = require('./src/008.js');
```

Comment:
Is this maybe left behind from testing, or does it do something?

Reply:
leftover, removing!